app-server/doky-front/src/components/Logo/Logo.jsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+const Logo = () => {
+  return (
+    <>
+      <img className='mb-3 mt-3 img-fluid' alt="Logo" src='/logo-color-bg.svg'/>
+    </>
+  );
+};
+
+export default Logo;
```

Comment:
JFYI: Probably overuse of default exports is not so good.

https://dev.to/phuocng/avoid-using-default-exports-a1c 👀